carma-messenger-core/docker/checkout.bash

```diff
@@ -36,7 +36,7 @@ while [[ $# -gt 0 ]]; do
 done
 
 if [[ "$BRANCH" = "develop" ]]; then
-      git clone https://github.com/usdot-fhwa-stol/carma-msgs.git ~/src/CARMAMsgs --branch $BRANCH --depth 1
+      git clone https://github.com/usdot-fhwa-stol/carma-msgs.git ~/src/CARMAMsgs --branch integration/geofence_msg --depth 1
```

Comment:
This should be left as $BRANCH having this change means we need to merge carma-msgs first.